tests/c/interp_inside_call2.c

```diff
@@ -0,0 +1,64 @@
+// Run-time:
+//   env-var: YKD_PRINT_IR=aot,jit-pre-opt
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     jit-state: start-tracing
+//     ...
+//     jit-state: stop-tracing
+//     ...
+//     jit-state: enter-jit-code
+//     ...
+//     jit-state: stopgap
```

Comment:
I wonder if we could expose more of the stopgap interpreter's workings for testing purposes?
Something like a `YKD_PRINT_SG_*` to print stuff like: the initial frames, maybe the instructions as they are interpreted? Overkill?

I just wonder if we are really testing much here.

Reply:
We could, but then we'd also have to add different lang tests for different O-levels. In this tests for example, for anything >= O1, the conditional branch is moved to the root frame, so there's only one initial frame.

But no matter what the optimiser does, this test will check that we can guard fail out of a nested call, so I think that's fine.